npm/vue/src/index.ts

```diff
@@ -14,8 +15,7 @@ const defaultOptions: (keyof MountOptions)[] = [
 ]
 
 function checkMountModeEnabled () {
-  // @ts-ignore
-  if (Cypress.spec.specType !== 'component') {
+  if (!Cypress.spec.relative.includes('cypress/component')) {
```

Comment:
Possibly... this is likely irrelevant now, since `mount` will only be used in the CT anyway.

I am also not sure we have access to config here? :thinking: either way I think we likely should be checking this higher up - we should most likely not do `mount` unless we are un the CT runner in the first place. 

The entire Vue `mount` function is due for a big rewrite this sprint: https://cypress-io.atlassian.net/browse/CT-237, so I will keep this in mind.